scripts/integration

```diff
@@ -4,30 +4,10 @@ function echo_with_time {
     echo "$(date --utc +%Y-%m-%dT%H:%M:%SZ) "$@""
```

Comment:
Again I know it's not part of this change, but I would change `"$@"` to `$*` because it's just for an `echo` statement. And `$@` outside double-quotes (which is what's going here) is the same as `$*`.